Formula/px.rb

```diff
@@ -0,0 +1,35 @@
+class Px < Formula
+  include Language::Python::Virtualenv
+
+  desc "Ps and top for human beings (px / ptop)"
+  homepage "https://github.com/walles/px"
+  url "https://github.com/walles/px.git",
+    using:    :git,
+    tag:      "1.4.0",
+    revision: "aea41c2a67a8b83e644e11d8ac152e2c123d0ee1"
+  license "MIT"
+
+  depends_on "python@3.9"
+
+  uses_from_macos "lsof"
+
+  # For updates: https://pypi.org/project/six/
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
+    sha256 "1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
+  end
+
+  # For updates: https://pypi.org/project/python-dateutil/#files
+  resource "python-dateutil" do
+    url "https://files.pythonhosted.org/packages/be/ed/5bbc91f03fa4c839c4c7360375da77f9659af5f7086b7a7bdda65771c8e0/python-dateutil-2.8.1.tar.gz"
+    sha256 "73ebfe9dbf22e832286dafa60473e4cd239f8592f699aa5adaf10050e6e1823c"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    assert_equal version.to_s, pipe_output("#{bin}/px --version").strip
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula